src/ngx_ssl_fingerprint.c

```diff
@@ -86,7 +86,7 @@ int ngx_ssl_fingerprint(ngx_connection_t *c, ngx_pool_t *pool, ngx_str_t *finger
     }
 
 
-    n = 33 + c->ssl->ciphers.len * 3 + c->ssl->extensions.len * 3 + c->ssl->groups.len * 6 + c->ssl->points.len * 4;
+    n = 32 + c->ssl->ciphers.len * 6 + c->ssl->extensions.len * 6 + c->ssl->groups.len * 6 + c->ssl->points.len * 4;
```

Comment:
`c->ssl->ciphers.len * 3 + c->ssl->extensions.len * 3` is enough, because openssl `SSL_Ctrl` for ciphers and extension already multiple 2.

1. https://github.com/openssl/openssl/blob/OpenSSL_1_1_1-stable/ssl/ssl_lib.c#L5402
2. https://github.com/openssl/openssl/blob/0a10825a009c830125fef94c81d34e41300a24a5/ssl/ssl_local.h#L460
3. https://github.com/phuslu/nginx-ssl-fingerprint/blob/master/patches/openssl.1_1_1.patch#L66